addons/web/static/src/js/libs/daterangepicker.js

```diff
@@ -10,5 +10,18 @@ daterangepicker.prototype.clickDate = function (ev) {
         clickDateFunction.apply(this, arguments);
     }
 };
+/**
+ * Compute daterangepicker to open up or down based on top/bottom space in window.
+ */
+const moveFunction = daterangepicker.prototype.move;
+daterangepicker.prototype.move = function (ev) {
+    const offset = this.element.offset();
+    if (offset.top + this.container.height() * 1.5 >= $(window).height() + $(window).scrollTop() && this.container.height() + this.element.outerHeight() < offset.top) {
```

Comment:
Could you explain why `this.container.height() < offset.top` isn't enough?

Reply:
I don't remember why I did like this :)